src/examples/test-and-release.yml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2018, salesforce.com, inc.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+# For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+
+description: >
+  Test npm package agaisnt node@12 and node@latest and then publish the package to npm
+usage:
+  version: 2.1
+  orbs:
+    release-management: salesforce/npm-release-management@x.y.z
+
+  workflows:
+    version: 2
+    test-and-release:
+      jobs:
+        - release-management/test-package:
+            name: node-latest
+            node_version: latest
+        - release-management/test-package:
+            name: node-12
+        - release-management/release-package:
+            context: AWS
```

Comment:
What is this? I don't see a context parameter in the orb.

Reply:
Contexts let you define a set of environment variables https://circleci.com/docs/2.0/contexts/